### ISSUE TITLE
wrong chaining logic

### ISSUE THREAD

**Issue by gglusman** [NCATSTranslator/Feedback#348]
Querying "what drugs may treat Postaxial Acrofacial Dysostosis (MIller Syndrome), the 2nd result is Brequinar, with two supporting paths. The second path chains 'brequinar causes decreased activity of DHODH' and 'DHODH causes Miller Syndrome'.
The problem is that the syndrome arises from disrupted DHODH activity (e.g., compound heterozygous null alleles), so it makes no sense to attempt to lower the activity of the gene.
One could hypothesize that a baby exposed to brequinar might develop Miller Syndrome!
There seems to be an assumption that genes need to be knocked down to treat diseases related to them.

**Comment by cbizon** [comment]
This is fundamentally a data problem - unless we have access to statements like "decreased DHODH causes Miller Syndrome" we have to make a guess if we want to use genetics at all.

Are there any KPs who want to rise to the challenge?  It would be a real game changer IMO.

**Comment by cbizon** [comment]
Another clarification ... I don't know which ARA produced this, but to my knowledge most of them are not using "logic" per se.  That is, the paths that are being returned do not prove in any way that the treats edge is correct.  Only that there is some evidence for the hypothesis.

**Comment by GitHubbit** [comment]
[https://medlineplus.gov/genetics/condition/miller-syndrome/#causes](url)

"Causes" section states: "Mutations in the [DHODH](https://medlineplus.gov/genetics/gene/dhodh/) gene cause Miller syndrome."

MedlinePlus appears to have a Genetic Conditions tab with this data. 

I am not sure if this is the most appropriate database for parsing this type of information out. What are other people's thoughts?
DisGeNet?
-Kamileh

**Comment by cbizon** [comment]
Well, the problem with that causes statement is that it doesn't really tell you what kind of mutations.  Especially if the causal mutations are missense, you don't really know if those are increase or decrease type mutations.

**Comment by GitHubbit** [comment]
So ClinVar then...
![image](https://github.com/NCATSTranslator/Feedback/assets/10294502/11b57490-8a4e-430b-af12-3331d9c16b67)

**Comment by cbizon** [comment]
Yeah, I think that in this case, because the variants are frameshifts you could make a directional assertion.

**Comment by gglusman** [comment]
Even frameshifts could lead to activation in special situations... a frameshift is not guaranteed to be a KO.
Kinda frustrating that even OMIM doesn't clearly state that Miller Syndrome involves a reduction in activity... https://omim.org/entry/263750

**Comment by gglusman** [comment]
Noting that a synonym of Miller Syndrome is "Dihydroorotate dehydrogenase **deficiency**"...
http://www.iembase.com/disorder/155

**Comment by gglusman** [comment]
@flannick Are there any databases clearly linking gene to disease and stating whether the effect is caused by KO, by deficiency, by disregulation, by overexpression...?

**Comment by mbrush** [comment]
Agree that "gene direction - causes - disease" type statements would be game changing for us. 
 
@cbizon  - might be worth asking your colleagues at ClinGen if they have access to anything like this, have thought about creating such a resource, or have ideas/data they could share with us that would help us create such a resource?
 
It seems like we/they could spin some kind of prediction service for this type of statement - given access to data we already have available to us: 
- we have knowledge about specific variants that cause disease, e.g. from ClinVar ("Variant x is pathogenic for Disease Y")
- we have info about the molecular consequence of these variants ("Variant X has molecular consequence Y")
- we have computational predictions about the functional impact of these variants ("Variant X has functional impact Y")
 - we have knowledge about domain architecture and functional motifs in proteins that govern its activity / regulation 
 
Even with just this knowledge as input, we could probably generate some pretty solid predictions about gene direction causing disease.  If we look at all variants in Gene X that are asserted to be pathogenic for Disease X in  ClinVar, and consider the molecular consequences and predicted functional impacts of the variants - an algorithm could make a pretty good guess about whether it is downregulation of the gene that is causal for the disease.  
- for example, if most/all pathogenic mutations are frameshifts for example, then decreased activity of the gene is likely causal for the disease.  
- but if we wee a bunch of frameshift mutations asserted to be benign, and only a handful of missense mutations clustered in specific areas of the gene are pathogenic, or truncations that impact regulatory regions of the protein - then we might predict that activating mutations.
 
There are likely other types of input that could inform such a model. And I'm sure its more complicated than this or it would have been done already.  But figured its worth thinking about.  

It may be that any prediction tool we might make would not be accurate enough to warrant making gene direction - causes - disease edges in our KGs explicitly, but could provide input to our evidence / result scoring algorithms.

**Comment by cbizon** [comment]
Yep, I have a meeting on Wed with Bradford & hope to find out more then.

**Comment by gglusman** [comment]
This issue was the topic of a session that @flannick organized in the February relay: https://drive.google.com/drive/u/1/folders/1-ftUazrzCLj-ppUBsrrrmIMro2Fdy22m

**Comment by flannick** [comment]
Interesting -- yes. I think that our KP should take this on.

I think that the ideas @mbrush lays out are right on. I think there are some annotations in gencc that indicate direction of effect, but this won't be comprehensive. We can dig a bit and see what we can get.

For *common* diseases, we already have some of these data in our KP, but not for the rare diseases. @marcdubybroad can you update our gencc/clinvar edges to add direction of effect where possible? We can discuss what a draft set of rules can be

**Comment by cbizon** [comment]
FWIW, It doesn't sound like ClinGen has exactly what we want either.  There is some information about genetic mechanism of disease, but it's probably not very well structured.  Now, it could perhaps be used with sufficient curation.

There are some parts of ClinGen that might help?  There is some gene dosage information that could probably be used to infer some cases, as well as variant annotations that could also be used to infer specific cases.    We're going to be working with the ClinGen folks to see what we can reasonably bring in.

**Comment by sierra-moxon** [comment]
from TAQA:
from GG: this is the topic for a February relay session from @flannick.  
up/down "amount" of a mutation of a gene causing or preventing a phenotype - computing from QTLs, etc. is hard, but there are papers about this (e.g. complete knockout).  (perhaps model organism dbs would be helpful here). 
should we crowdsource curated assertions?  
but higher level - Translator missing this information means we're returning answers that are opposite of what was asked for. 

Tyler: we might not have time for this before September if it means bringing in new data sources. 
Sui: we have datasets that can help - the assertions are correct, but the labels are wrong.  Biolink needs "knockout" permissible value for a "subject_form_or_variant_qualifier" -- there could be some complication here, a qualifier of a qualifier. 
Sierra is talking to Brett about getting this added to Biolink in v3.5.0 for September.
Matt: we could use the underlying data to come up with rules that allow us to automatically assign predictions to variants.  (VEP might also do this).

**Comment by sierra-moxon** [comment]
from TaQA:
Sui: direction of effect and variant affect might be separable issues. 
Chris: September aside - this is IMO an important point of data that we need a KP to bring in.

**Comment by sandrine-m** [comment]
After meeting at the DM call today; those type of data gap/reasoning issues need more work and will be fixed post September.

**Comment by sierra-moxon** [comment]
This is a priority for review from TACT (retrieving directional data from gene to disease for example).

**Comment by mbrush** [comment]
Adding text from a post @gglusman made to the #general Slack channel on Oct 5 - which was reported to be relevant to this issue:

> While we're sharing literature, "[Literature-based predictions of Mendelian disease therapies](https://www.cell.com/ajhg/fulltext/S0002-9297(23)00313-0)" is of interest in several ways. The "PARMESAN" tool they describe extracts from literature relationships between entities with modifiers for direction of effect - a topic we discussed in the previous relay and is still an open issue for Translator. Table 1 lists the terms used to distinguish between 'positive' and 'negative' predicates. They then predict indirect relationships, not unlike our creative mode. They evaluate the results in various ways, including a comparison with SemMedDB.

> The database of predicted relationships can be [queried online](https://parmesan.nrihub.org/#/home) or downloaded in bulk (a 1.5 GB download). It includes a bunch of TSVs that could be quite useful, beyond the predictions themselves...
